### PR TITLE
RENO-1829 - center alert/closing message in global header

### PR DESCRIPTION
**Pull request by knezmilos**
Centers the alert/closing message in global header.  It also removes margins on 2 breakpoints so the message is always centered

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
Can you update the changelog and the version in package.json to 2.7.1?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ### CHANGE LOG
 
 ========
+### v2.7.1
+> Updates packages for a better build.
```
Comment (EdwinGuzman) [inline CHANGELOG.md:5]: This is from the previous version so it can be removed here.

**Review by EdwinGuzman — APPROVED** [review]
Looks good but the change log needs a minor update.

**Comment by knezmilos** [comment]
Changelog updated

**Review by EdwinGuzman — APPROVED** [review]

**Comment by EdwinGuzman** [comment]
@knezmilos @jberube-nypl version `2.7.1` is now published on npm.

**Pull request merged**